Task1/Patryk/main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <cstdio>
+
+using namespace std;
+
+bool isPrime(int value)
+{
+    bool isPrimeInd = true;
+    int i = 2;
+    do
+    {
+        if ((value % i == 0 || value < 2) && value!=2)
+        {
+            isPrimeInd = false;
+            break;
+		}
+        i++;
+    }while (i<value);
+	return isPrimeInd;
+}
+
+void primeChecker(int value)
+{
+    if(isPrime(value))
+    {
+        cout << value << " is prime!" << endl;
+    }
+    else
+        cout << value << " is not prime!" << endl;
+}
+
+void listOfPrimes(int value)
+{
+    cout << "All primes that are less than " << value <<endl;
+    for(int i=1; i<value; i++)
+    {
+        if(isPrime(i))
+        {
+                cout << i << "  ";
+        }
+    }
+}
+
+int main()
```

Comment:
overall it's nicely written little program. Logic is correctly organised into smaller functions with sensible names and arguments. I'm quite happy with this program and would recommend it to my friends